content/ai/concepts/neural-networks/terms/activation-function/activation-function.md

```diff
@@ -0,0 +1,30 @@
+---
+Title: 'Activation Function'
+Description: 'An activation function is the function used by a node in a neural network to take the summed weighted input to the node and transform it into the output value.'
+Subjects:
+  - 'Machine Learning'
+  - 'Data Science'
+  - 'Computer Science'
+Tags:
+  - 'AI'
+  - 'Deep Learning'
+  - 'Machine Learning'
+  - 'Neural Networks'
+  - 'Algorithms'
+  - 'Functions'
+CatalogContent:
+  - 'machine-learning'
+  - 'paths/data-science'
+---
+
+An **activation function** is the function used by a node in a neural network to take the summed weighted input to the node and transform it into the output value. The output value can be fed into the next hidden layer or received from the output layer. For networks that engage in tasks of any significant complexity, the activation function needs to be non-linear.
+
+## Types of Activation Functions
```

Comment:
I believe the "relu" is one the most commonly used functions and probably should be mentioned here as well